### PR TITLE
Added Api contract test case for profiles endpoint

### PR DESCRIPTION
**Pull request by gokulakrishnansvm**
Adding API Contract Test Case for profiles endpoint.
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops Test

## What is the best way to verify this PR?

Step 1 : Install requirements
pip install -r requirements.txt

Step 2 : To execute profiles API Contract Testcase , navigate to traffic_ops/testing/api_contract/v4
pytest --to-user Username --to-password Password --to-url URL --config Config Path --request-template Request template --response-template Response template test_profiles.py

Note: Config, Request template and response template paths are optional. 

## PR submission checklist
- [x] This PR has tests 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_ops/testing/api_contract/v4/test_profiles.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	request_template_data: list[dict[str, object] | list[object] | primitive],
+	response_template_data: dict[str, primitive | list[primitive | dict[str, object]
+						    | list[object]] | dict[object, object]],
+	profile_post_data: dict[str, object]
+) -> None:
+	"""
+	Test step to validate keys, values and data types from profiles endpoint
+	response.
+	:param to_session: Fixture to get Traffic Ops session.
+	:param request_template_data: Fixture to get profile data from a prerequisites file.
```
Comment (ericholguin) [inline traffic_ops/testing/api_contract/v4/test_profiles.py:39]: missing response_template_data

**Review by ericholguin — CHANGES_REQUESTED** [review]
Minor change, everything else looks good. Tests are passing

File: traffic_ops/testing/api_contract/v4/test_profiles.py (hunk 1 above)
Comment (gokulakrishnansvm) [reply traffic_ops/testing/api_contract/v4/test_profiles.py]: Update it!

**Review by ericholguin — APPROVED** [review]
LGTM

**Review by TerekhovaKate — APPROVED** [review]

**Pull request merged**